docs/source/getting-started/Setting-up-hypervisor-driver.adoc

```diff
@@ -1,20 +1,20 @@
-[[docker-machine-driver-install]]
-= Installing Docker Machine drivers
+[[set-up-hypervisor-driver]]
```

Comment:
The order of using `setup` or `set-up` in the front of the filename is an improvement of having "install" at the end. 

Browsing the source tree for the docs from the computer or https://github.com/minishift/minishift/tree/master/docs/source is non-intuitive. 

Before:
![install1](https://cloud.githubusercontent.com/assets/1894/26415640/ce2d9e6a-40e5-11e7-877c-73e2dbc7eae2.png)

After:
![install2](https://cloud.githubusercontent.com/assets/1894/26415648/d34036ce-40e5-11e7-81b9-2ba7491fe168.png)

While I agree that the uppercase is not correct, the name indicates more meaning... `docker-machine-drivers.adoc` not so much